From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 15 Dec 2014 22:34:53 +0000
Subject: iovec: Fix ABI change in 3.16.7-ckt2-1
Forwarded: not-needed

I got confused and tried to fix an ABI change from upstream changes
that we already had.  Thanks to #773233 this wasn't spotted before
upload.  Keep the extra function around.

--- a/mm/iov_iter.c
+++ b/mm/iov_iter.c
@@ -725,6 +725,15 @@ ssize_t iov_iter_get_pages(struct iov_it
 }
 EXPORT_SYMBOL(iov_iter_get_pages);
 
+/* bwh: Alias for iov_iter_get_pages() because I'm an idiot */
+ssize_t iov_iter_get_pages_fixed(struct iov_iter *i,
+		   struct page **pages, size_t maxsize, unsigned maxpages,
+		   size_t *start)
+{
+	return iov_iter_get_pages(i, pages, maxsize, maxpages, start);
+}
+EXPORT_SYMBOL(iov_iter_get_pages_fixed);
+
 ssize_t iov_iter_get_pages_alloc(struct iov_iter *i,
 		   struct page ***pages, size_t maxsize,
 		   size_t *start)
